critique.py

```diff
@@ -153,13 +256,14 @@ def walk_function(registrar, path, files):
     if path.find('.git') > -1:      # skip the Git VCS directory
         return
     for nm in files:
```

Comment:
extensions = ['.txt', '.py', '.rst', '.md', '.in']

Reply:
I'll call the variable skip_extensions since I think that makes the intent more clear.